ikura/chain/runtimes/gondatsu/src/lib.rs

```diff
@@ -120,7 +120,7 @@ impl_opaque_keys! {
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("blobchain-kusama"),
-    impl_name: create_runtime_str!("blobchain-kusama"),
+    impl_name: create_runtime_str!("gondatsu"),
```

Comment:
https://paritytech.github.io/polkadot-sdk/master/frame/runtime/prelude/struct.RuntimeVersion.html#structfield.impl_name

Reply:
"Name of the implementation of the spec. This is of little consequence for the node and serves only to differentiate code of different implementation teams"